classifier/README.md

```diff
@@ -1 +1,57 @@
+# Summary Quality Classifier
+
+This classifier assesses the quality of text summarization result in terms 
+of grammar and meaning preservation. This model modified from the 
+[LaserTagger](https://github.com/google-research/lasertagger) script, and 
+is built on Python 3, Tensorflow and 
+[BERT](https://github.com/google-research/bert). 
+
+BERT transformer model is used as the encoder, and for the decoder, a single
+feed-forward pass is applied to the decoder. This model should be trained on
+the 
+[Microsoft Abstractive Text Compression Dataset](https://www.microsoft.com/en-us/download/confirmation.aspx?id=54262).
+This dataset contains ratings for each source-summary pair, and is scored on
+grammar and meaning on a 0-2 scale. 
```

Comment:
why 0-2 not 0-1 scale?

Reply:
The dataset has three levels of scoring for grammar and for meaning. I am not sure why they did this. I converted the scoring into 0, 1, 2 in preprocessing.